cloudformation/028-cfn-splash-page.yaml

```
@@ -0,0 +1,441 @@
+    Properties:
+      Name: !Sub "internal-${ServiceName}"
+      HealthCheckIntervalSeconds: 60
+      HealthCheckPath: "/version"
+      HealthCheckTimeoutSeconds: 5
+      UnhealthyThresholdCount: 5
+      HealthyThresholdCount: 2
+      Port: 8182
+      Protocol: HTTP
+      TargetType: ip
+      VpcId: !ImportValue prod-vpc-PrivateVpcId
+
+      Tags:
+        - Key: ProjectName
+          Value: !Ref ProjectName
+
+  fargateSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: security group applied to containers
+      VpcId: !ImportValue prod-vpc-PrivateVpcId
+      SecurityGroupIngress:
+        - IpProtocol: 'tcp'
+          FromPort: 80
+          ToPort: 80
+          SourceSecurityGroupId: !ImportValue prod-general-infra-Lb-SecurityGroupId
+          Description: Allow access from external facing lb to api auth sidecar
+        - IpProtocol: 'tcp'
+          FromPort: 80
+          ToPort: 80
+          SourceSecurityGroupId: !ImportValue prod-general-infra-InternalLb-SecurityGroupId
+          Description: Allow access from internal facing lb to api auth sidecar
+        - IpProtocol: 'tcp'
+          FromPort: 8182
+          ToPort: 8182
+          SourceSecurityGroupId: !ImportValue prod-general-infra-Lb-SecurityGroupId
+          Description: Allow access from external facing lb to app after browser authentication
+        - IpProtocol: 'tcp'
+          FromPort: 8182
+          ToPort: 8182
+          SourceSecurityGroupId: !ImportValue prod-general-infra-InternalLb-SecurityGroupId
+          Description: Allow access from internal facing lb to app without auth
+      SecurityGroupEgress:
+        - FromPort: 443
+          ToPort: 443
+          IpProtocol: 'tcp'
+          CidrIp: '0.0.0.0/0'
+          Description: Needs to allow access to cognito endpoint for jwt stuff and there is no vpc endpoint yet
+        - FromPort: 443
+          ToPort: 443
+          IpProtocol: 'tcp'
+          DestinationPrefixListId: !Ref S3PrefixListId
+          Description: Allow outbound to S3
+        - FromPort: 80
+          ToPort: 80
+          IpProtocol: 'tcp'
+          DestinationPrefixListId: !Ref S3PrefixListId
+          Description: Allow outbound to S3
+      Tags:
+        - Key: ProjectName
+          Value: !Ref ProjectName
+
+  listenerRuleExternalBrowser:
+    Type: AWS::ElasticLoadBalancingV2::ListenerRule
+    Properties:
+      Actions:
+        - Type: authenticate-cognito
+          Order: 1
+          AuthenticateCognitoConfig: 
+            OnUnauthenticatedRequest: authenticate
+            Scope: openid
+            SessionCookieName: AWSELBAuthSessionCookie
+            SessionTimeout: 28800
+            UserPoolArn: !ImportValue prod-auth-Cognito-UserPoolArn
+            UserPoolDomain: !ImportValue prod-auth-Cognito-UserPoolDomain
+            UserPoolClientId: !Ref cognitoAppClient
+        - TargetGroupArn: !Ref targetGroupExternalBrowser
+          Order: 2
```

```suggestion
          Order: 1
```
Are we wanting the spash page do be behind a log-in wall? If not change this, if so leave it be.